modules/custom/az_paragraphs/css/az_paragraphs_az_text_media.css

```
@@ -27,11 +27,14 @@
   background: none;
 }
 .box {
-  padding: 2em;
-  margin: 3em 0;
+  padding-left:2em;
+  padding-right: 2em;
+  margin-left: 0;
+  margin-right:0;
```

Actually, since we aren't altering the margin fo box we shouldn't touch it here.
```suggestion
  padding-left:2em;
  padding-right: 2em;
  margin: 3em 0;
```